src/components/PostDetails.tsx

```diff
@@ -1,117 +1,98 @@
 import React from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
+import { ErrorType } from '../types/ErrorType';
+import { CommentsList } from './CommentList';
+
+type Props = {
+  post: Post | null,
+  comments: Comment[],
+  errorType: ErrorType | null,
+  setErrorType: React.Dispatch<React.SetStateAction<ErrorType | null>>,
+  setComments: React.Dispatch<React.SetStateAction<Comment[]>>,
+  isFetch: boolean,
+  isCommentFormOpen: boolean,
+  openFormForComment: React.Dispatch<React.SetStateAction<boolean>>,
+};
+
+export const PostDetails: React.FC<Props> = ({
+  post,
+  comments,
+  errorType,
+  setErrorType,
+  setComments,
+  isFetch,
+  isCommentFormOpen,
+  openFormForComment,
+}) => {
+  const handleOpenCommentForm = () => {
+    openFormForComment(true);
+  };
+
+  const errorTypes: ErrorType[] = [
```

Comment:
The variable 'errorTypes' is declared but its value is never read. If this variable is not used, consider removing it to improve code readability.